.github/workflows/website.yaml

```diff
@@ -0,0 +1,31 @@
+name: Website deploy
+on:
+  push:
+    branches:
+    - stable
+
+jobs:
+  website-nightly:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: true
+    - uses: actions/setup-node@v2.5.0
+      if: env.SHOULD_PUBLISH == 'true'
```

Comment:
```suggestion
```
I don't think we need this `if` because we always want to deploy docs on this branch.